packages/scene-composer/src/components/panels/scene-components/AnimationComponentEditor.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback } from 'react';
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import { useStore } from '../../../store';
+import { mockNode, mockProvider } from '../../../../tests/components/panels/scene-components/MockComponents';
+import { generateUUID } from '../../../utils/mathUtils';
+
+import { AnimationComponentEditor } from './AnimationComponentEditor';
+
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useCallback: jest.fn(),
+}));
+
```

Comment:
do not mock react hooks, but instead you should interact with the component (like fire click event) to trigger the callback